Dockerfile

```
@@ -0,0 +1,9 @@
+FROM node:12
```

```suggestion
FROM node:14-alpine
```
1. Node version 12 will be end-of-life pretty soon. Node version 14 is Active LTS right now and will [very] soon be in Maintenance LTS. 
I think we should go for *at least* 14, if not 16 since 16 will [very] soon be the Active LTS. https://nodejs.org/en/about/releases/ 
I tested with Node 14 and it seems fine, no fires. Supposedly anything >= 12.0.0 should work for apostrophe 3 https://github.com/apostrophecms/apostrophe/blob/1e597f097af4fccff444e3bd085b435a6943818b/package.json#L16
2. Alpine is great because it's about 10% of the size of the regular image